dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -32,6 +32,7 @@ REST_FORBIDDEN=Forbidden
 REST_INVALID_CONTENT_TYPE=The request's content-type header was not ''application/json''.
 REST_MISSING_OPERATION=The operation was not specified for the request.
 REST_NO_ACME_SERVICE=An AcmeProvider service was not registered.
+REST_TOO_MANY_REQUESTS=The user has sent too many requests in a given amount of time. The amount of time remaining until the next request is allowed is {0}.
```

Comment:
**change**
The user has sent too many requests in a given amount of time. The amount of time remaining until the next request is allowed is {0}.
**to**
The user sent too many requests in a given amount of time. The amount of time remaining until the next request is allowed is {0}.

Reply:
Done